packages/twenty-server/src/workspace/workspace-datasource/workspace-datasource.service.ts

```diff
@@ -114,16 +115,18 @@ export class WorkspaceDataSourceService {
     workspaceId: string,
     transactionManager?: EntityManager,
   ): Promise<any> {
-    if (transactionManager) {
-      return await transactionManager.query(query, parameters);
-    }
-    const workspaceDataSource =
-      await this.connectToWorkspaceDataSource(workspaceId);
+    try {
+      if (transactionManager) {
+        return await transactionManager.query(query, parameters);
+      }
+      const workspaceDataSource =
+        await this.connectToWorkspaceDataSource(workspaceId);
 
-    if (workspaceDataSource) {
       return await workspaceDataSource.query(query, parameters);
+    } catch (error) {
+      throw new Error(
```

Comment:
@Weiko Hmm not sure if it's the right level to catch the error, we should let query throw and let the exception handlers transform the errors properly, as we're also mapping some of the `pg_graphql` and `postgres` errors